fe/src/main/java/org/apache/doris/catalog/MaterializedIndexMeta.java

```diff
@@ -46,9 +55,11 @@
     private TStorageType storageType;
     @SerializedName(value = "keysType")
     private KeysType keysType;
+    @SerializedName(value = "defineStmt")
+    private OriginStatement defineStmt;
```

Comment:
The name defineStmt can easily cause ambiguity with DefineExpr. But this statement may not only parse the semantics of defineExpr in the future。MaterializedViewStmt is a better name ?

Reply:
The define stmt means that this is the definition of materialized views.